### PR TITLE
Update DeploymentConfig controller to use getPreferredVersion

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Depends on [web-common PR 201](https://github.com/openshift/origin-web-common/pull/201) and [web-common PR 202](https://github.com/openshift/origin-web-common/pull/202)

### CODE REVIEW

File: app/views/browse/deployment-config.html [hunk 1]
```diff
@@ -8,7 +8,7 @@ <h1 class="contains-actions">
           <div class="pull-right dropdown" ng-if="deploymentConfig" ng-hide="!('deploymentConfigs' | canIDoAny)">
```
Comment (spadgett) [inline app/views/browse/deployment-config.html]: We should be updating the `canIDoAny` checks to use the same resource versions as the actions

**Comment by spadgett** [comment]
We'll also want to update the health check URL to use the group from preferred version on this PR

**Comment by benjaminapetersen** [comment]
Just rebased & caught this one up w/the other PRs re: `healthCheckUrl` & `canIDoAny`.

@spadgett

**Comment by benjaminapetersen** [comment]
rebased

**Comment by spadgett** [comment]
[merge]

**Pull request merged**